frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -585,13 +585,9 @@ describe("scenarios > dashboard", () => {
         cy.get("fieldset")
           .contains("Category")
           .click();
+        cy.findByPlaceholderText("Search the list").type("Gizmo");
```

Comment:
It seems we changed the placeholder wording.

Reply:
No, it was just filter dropdowns that had been broken on dashboards for a long time. https://github.com/metabase/metabase/pull/15138#issuecomment-797794099